test/python/docker/compat/constant.py

```diff
@@ -1,5 +1,5 @@
 """
-Constants to use in writing unittests.
+Constants to use in writing unit tests.
```

Comment:
Looks good but I should've pointed out this other python reference to one-word unittests. Could you do a quick repush? (There's one other "unittests", above, but that's Go and I think your edit is correct). Sorry for not pointing this one out.